modules/apps/change-tracking/change-tracking-web/build.gradle

```diff
@@ -15,6 +15,7 @@ dependencies {
 	compileOnly project(":apps:change-tracking:change-tracking-spi")
 	compileOnly project(":apps:configuration-admin:configuration-admin-api")
 	compileOnly project(":apps:diff:diff-api")
+	compileOnly project(":apps:dynamic-data-mapping:dynamic-data-mapping-api")
```

Comment:
@gislayne-vitorino is this dependency needed?

Reply:
No, looks like it's not needed anymore. will send an update now